packages/react/src/components/dynamic-flow/handle-form.tsx

```diff
@@ -105,6 +106,27 @@ export const HandleForm: React.FC<Props> = ({
       );
     }
 
+    if (handleType === "username") {
+      return (
+        <Input
+          className={sprinkles({ marginTop: "4" })}
+          id={`sid-input-${handleType}`}
+          name={handleType}
+          label={text["initial.handle.username"]}
+          placeholder={text["initial.handle.username.placeholder"]}
+          value={values[handleType] ?? ""}
+          onChange={registerField(handleType, {
+            defaultValue,
+            validator: (value) => {
+              if (!isValidUsername(value)) {
+                return { message: text["validationError.email"] };
```

Comment:
Should probably use a different validation error

Reply:
sure, thanks!

Reply:
fixed